lib/WebService/GrowthBook/FeatureResult.pm

```diff
@@ -4,25 +4,23 @@ use warnings;
 no indirect;
 use Object::Pad;
 use Log::Any qw($log);
+use JSON::MaybeXS;
 
 ## VERSION
 
 class WebService::GrowthBook::FeatureResult{
     field $id :param :reader;
     field $value :param :reader;
-    field $type :param :reader;
     method on{
-        if ($type eq 'boolean'){
-            return $value ? 1 : 0;
+        if (JSON::MaybeXS::is_bool($value)){
+            return $value;
         }
-        $log->errorf("FeatureResult->on called on non-boolean feature %s", $id);
+        $log->errorf("FeatureResult->on/off called on non-boolean feature %s", $id);
         return undef;
     }
     method off{
-        if ($type eq 'boolean'){
-            return $value ? 0 : 1;
-        }
-        $log->errorf("FeatureResult->off called on non-boolean feature %s", $id);
-        return undef;
+        my $result = $self->on;
+        return $result unless defined($result);
+        return $result ? JSON::MaybeXS::false : JSON::MaybeXS::true;
```

Comment:
I think it would be better to just return value as 0 or 1, and make it consistent for `on` [method](https://github.com/deriv-com/perl-WebService-GrowthBook/pull/4/files#diff-6751fdfad83f2ea2296ba3aa7438f4aeeaabd84594997b2343942a2ff54e5cddR16) too.